honeybee_vtk/vtkjs_helper.py

```diff
@@ -0,0 +1,213 @@
+import base64
+import json
+import re
+import os
+import shutil
+import sys
+import zipfile
+
+try:
+  import zlib
+  compression = zipfile.ZIP_DEFLATED
+except:
+  compression = zipfile.ZIP_STORED
+
+# -----------------------------------------------------------------------------
+
+def convertDirectoryToZipFile(directoryPath):
```

Comment:
I know you didn't write this code but now that it is part of our code base it should follow our code style. Let's fix these changes.

```suggestion
def convert_directory_to_zip_file(directory_path):
```

Reply:
I made the quick changes that I could make. I believe we should prioritize testing this package more and writing more test. I would ideally like to remove this module from this package. A way to do that would be to build the vtk package but I have not yet figured out that fully.

Reply:
I'm not sure why would you want to remove it when it works? Let's just clean it up and use what they have done,.